VirtoCommerce.Platform.Web/VirtoCommercePlatformWebBootstrapper.cs

```diff
@@ -49,7 +51,7 @@ public override void Run(bool runWithDefaultConfiguration)
             base.Run(runWithDefaultConfiguration);
 
             //registering Unity for MVC
-            //DependencyResolver.SetResolver(new UnityDependencyResolver(container));
+            DependencyResolver.SetResolver(new MvcUnityDependencyResolver(Container));
```

Comment:
Is this change necessary?

Reply:
Yes, it enables DI for regular (non-WebAPI) MVC controllers. It is used to inject dependencies to the `ExternalLoginController`.